backend/utils/psyonix_api_handler.py

```diff
@@ -199,5 +202,6 @@ def get_platform_id(i):
     else:
         return '-1'
 
+
 if __name__ == '__main__':
-    print(get_rank('76561198374703623'))
\ No newline at end of file
+    print(get_rank_batch(['76561198374703623'], use_redis=False))
```

Comment:
move to a test file?

Reply:
Nope, that would require putting the RLAPI key in a test file.